public/js/components/stories/Frames.js

```diff
@@ -0,0 +1,32 @@
+const React = require("react");
+const { DOM: dom, createElement } = React;
+const { Provider } = require("react-redux");
+
+const { createStore } = require("./utils");
+
+const { storiesOf } = require("@kadira/storybook");
+const Frames = React.createFactory(require("../Frames"));
+
+const frameData = require("../../test/fixtures/frames.json");
+
+storiesOf("Frames", module)
+  .add("Blank", () => {
+    return renderContainer(Frames);
+  })
+  .add("Paused", () => {
+    return renderContainer(Frames, { frames: frameData });
+  });
+
+function renderContainer(Component, props) {
+  const store = createStore();
```

Comment:
I do because the Frames component still uses `connect` to get the action creator `selectSource`. That is confusing though.

I'm going to think some more about the pattern for "connecting" components. For now I might just make `Frames` pull the state off too because everything is doing that.